src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -135,18 +141,40 @@ export function NavGroups({
     }
 
     if (navLink.itemType === LinkItemType.PARENT_LINK && navLink.link) {
-      return {
-        ...createNavItem({ link: navLink.link }),
+      const props = createNavItem({ link: navLink.link });
+      const parentItem = {
+        ...props,
         forceOpen: true,
-        items: navLink.links.map((subNavLink) => createSideNavItem(subNavLink, 'nav-nested-item')),
+        href: undefined,
+        onClick: undefined,
+        className: classNames(props.className, 'nav-link-parent-item'),
+        buttonClassName: classNames(props.buttonClassName, 'nav-link-parent-item-button'),
+        items: navLink.links.map((subNavLink) =>
+          createSideNavItem(subNavLink, level + 1, 'nav-nested-item')
+        ),
       };
+      /**
+       * OuiSideBar will never render items of first level as accordion,
+       * in order to display accordion, we need to render a fake parent item.
```

Comment:
Is there a screenshot of the accordion?

Reply:
### Before:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/44e96663-ad8f-4ab0-820f-726813961499">

### After:
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/5002a56d-c809-47a3-a9f0-511df3e1445a">